distributed/tests/test_stress.py

```diff
@@ -284,3 +286,35 @@ async def test_no_delay_during_large_transfer(c, s, w):
     nbytes -= nbytes[0]
     assert nbytes.max() < (x_nbytes * 2) / 1e6
     assert nbytes[-1] < (x_nbytes * 1.2) / 1e6
+
+
+@gen_test()
+async def test_chaos_rechunk():
+    async with LocalCluster(
```

Comment:
Is there a reason why this needs to be a local cluster? I often need to run these tests a few hundred times in parallel. The LocalCluster has some ports hard coded

Reply:
I wanted to use adaptive.  I can work on getting this to a random port

Reply:
This should now be done.